ansible/playbook.yml

```
@@ -0,0 +1,199 @@
+        git_branch: "{{ env_vars.GIT_BRANCH | default(git_branch) }}"
+        allowed_hosts: "{{ env_vars.ALLOWED_HOSTS | default('localhost,127.0.0.1') }}"
+
+    - name: Ensure application user exists
+      user: { name: "{{ vps_user }}", system: yes, create_home: yes, shell: /bin/bash }
+
+    - name: Clone code
+      git:
+        repo: "{{ git_repo }}"
+        dest: "{{ project_root }}"
+        version: "{{ git_branch }}"
+        force: yes
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)